### DISCUSSION TITLE
How to lift or ignore @unique constraint for certain columns in demo mode? (SQLite connector)

### DISCUSSION THREAD

**Discussion by ridhwaans** [prisma/prisma#13826] [Q&A] [answered]
Suppose you want to fill in dummy data that isn't unique in a staging or demo environment. 
 
is there a clean solution to removing `@unique` constraints only in environments that aren't prod, without repeatedly running table migrations?

thanks

**Comment by austincrim** [comment, marked as answer]
Hey, @ridhwaans 👋, sorry for the delayed response.

It is generally not recommended to have disparate schemas per environment as it might introduce subtle and unexpected bugs if you are developing and testing against a schema that is not an accurate representation of your production environment.

It is possible to create multiple Prisma Schemas and [generate the clients to unique locations](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client#using-a-custom-output-path) to be used by your tests, but this would introduce some maintenance burden.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
